src/components/Premium/Menu.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+const Menu = () => {
+  return (
+    <St.MenuContainer>
+      <St.Button type = "button"> 전체 </St.Button>
+      <St.Button type = "button"> 오특 </St.Button>
+      <St.Button type = "button"> 신상 </St.Button>
+      <St.Button type = "button"> 랭킹 </St.Button>
+      <St.Button type = "button"> 기획전 </St.Button>
+      <St.Button type = "button"> 이벤트 </St.Button>
+    </St.MenuContainer>
+
+  );
+};
+
+export default Menu;
+
+const St = {
+  MenuContainer: styled.nav`
+    display: flex;
+    justify-content: space-evenly;
```

Comment:
요기 맨 좌우는 17px이고 메뉴간 간격이 9px라서
이 속성을 주면 맨 좌우까지 모두 동일한 간격이 되어용!!
```suggestion
    gap: 0.9rem //이렇게 간격 주고
    
    padding: 0 1.7rem; //좌우는 이렇게 줘도 되겠다 ㅎㅎ
```